.gitignore

```diff
@@ -3,4 +3,5 @@
 tmp/
 *-tmp.yaml
 user-dir/
-user-dir-*/
\ No newline at end of file
+user-dir-*/
+v4m-container/kubeconfig/
```

Comment:
Since docs recommend copying `kubeconfig` here, we should just ignore it from git to avoid accidental pushes of configs